packages/manager/apps/dedicated/client/app/account/user/translations/Messages_fr_FR.json

```diff
@@ -608,5 +608,7 @@
   "user_account_support_level_title": "Mon niveau de support",
   "user_account_ssh_keys_title": "Mes clés SSH",
   "user_account_users_management_title": "Gestion des utilisateurs",
-  "user_account_advanced_title": "Paramètres avancés"
+  "user_account_advanced_title": "Paramètres avancés",
+  "user_account_guides_header": "Guides",
```

Comment:
Same Question : https://github.com/ovh/manager/pull/7625/files#r927543368

Reply:
Yes, for example in spanish "guides" is said "Guías".